docs/components/admin-app/index.md

```diff
@@ -0,0 +1,34 @@
+# The Admin App
+
+If you have the [application running](../../deployment/quick-setup.md), you can access the admin app at:
+
+The Admin App allows you to do the following:
+
+    https://[DOMAIN]/
+
+or if you are using the [dev](../../develop/setup.md) setup:
+
+    http://localhost:3000/
+
+## Features
+
```

Comment:
Should we add access level here also? Otherwise we can only see this feature from left menu

Reply:
Also, can we have cards here like you did for Core Components index? Seems more consistent and easy to view!

Reply:
I can do cards. I was just being lazy 😛 

Re: access level - it's not really a capability. But I will mention it on this page so there is another reference to it